### ISSUE TITLE
> Owner - owner is a arbitrary contract (wallet or dao) that could make critical updates to the contract: change fees, start/stop pool and even update code

### ISSUE THREAD

**Issue by Tpkilgore197** [ton-foundation/ton-nominators#31]
> Owner - owner is a arbitrary contract (wallet or dao) that could make critical updates to the contract: change fees, start/stop pool and even update code

Does it mean that owner can update his contract with already nominated coins? What will prevent him from adding code that will dispose of coins of nominators?

_Originally posted by @jemeraldo in https://github.com/tonwhales/ton-nominators/issues/5_

**Comment by Tpkilgore197** [comment]
Seems someone one you side already saw that one before you and she went into my drive and deleted files also their is a half a million dollars missing.   And I don't need your help to try to keep my money from me and trust wallet removed the coin from my watch only wallet.  Seems that I can't remove them and spend them but they sure did I want a full and detailed report on all my trans actions with a full log of who and where from the start you can't let someone just steal my coin while I don't have access to move them that's like stealing from the blind or handicap I guess honor that the coders had has gone out the window and theives have moved in to take and watch over the internet we can't do it but it's ok for y'all to.   Naw

**Comment by Tpkilgore197** [comment]
Copy's of all of this aiccident is getting blasted all over the world to show what kinda folk we trust to handle our life savings.  And all the way to the security consultant at sec.   So get ready and put my money back I am withdrawing all of my my crypto in usd and getting away from it due to the dishonest people in a self honor and controlled investment environment.   You take advantage of people that you think are stupid well you got caught this the I also want to know where my money in coingecko portfolio is located and I way it with drawn all 94 million and market cap all 137 million into US dollars and place in a bank account because I am taking it and leaving the country I hate the way this place I fought for has turned out am I will just take my 200mill out of this country and see how well the economy grows then     You people better pray I don't and better figure out how to make me happy before I do start controlling your moves and firing every one of you

**Comment by Tpkilgore197** [comment]
Why don't you quit messing with my life and my opportunities of a better quality of life and sweep your own front porch I am a Native American served this country and have metals of valor an honorable discharge and am a disable very and only get 1700 a month to live on so yes I started trying to learn code and how to trade and mine coins as it stand you people have me starving nasty no shower or bath no place to piss nothing to eat and can't go anywhere.    So I am asking because I did well and served this county never have had a felony arrest I get treated inhumane as bad and I know way worse than murders baby rapers and converted felons and then have y'all trying to take away my life and my investments on top over that.    It's a hard one to swallow and I have been poor all my life once I finally get a chance to live a life God meant for me to have before I die and you people pedy theives try and steal it from me and give me an allowance of five thousand dollars.   You really think y'all are fair or just maybe y'all are the reason the world is fucked ip now.     Just leave my stuff alone and quit trying to take it you do not get to earn it and don't deserve it
